cmd/argocd/commands/utils/prompt.go

```diff
@@ -0,0 +1,30 @@
+package utils
+
+import (
+	"github.com/argoproj/argo-cd/v2/pkg/apiclient"
+	"github.com/argoproj/argo-cd/v2/util/cli"
+	"github.com/argoproj/argo-cd/v2/util/localconfig"
+)
+
+type Prompt struct {
+	enabled bool
+}
+
+func NewPrompt(clientOpts *apiclient.ClientOptions) (*Prompt, error) {
+	localCfg, err := localconfig.ReadLocalConfig(clientOpts.ConfigPath)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
Add override param on `clientOpts`

Reply:
I've added a global CLI flag `--force-prompts-enabled` that will override the value in local config if it's set